manifests/rabbitmq_save_conf.pp

```diff
@@ -18,9 +18,15 @@
     fail("Rabbitmq port not set.")
   }
 
-  $ctrl_ip = hiera('vrts_ctrl_mgmt_ip', '')
-  if $ctrl_ip == '' {
-    fail("vrts_ctrl_mgmt_ip IP not set.")
+  $mgmt_ips = hiera('vrts_config_param1', '')
```

Comment:
In case of non-HA setup, what will be specified in vrts_ctrl_mgmt_ip?
Will it work if the VIP is specified in vrts_ctrl_mgmt_ip? Is it tested?